modules/tinymce/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Typing after deleting formatted content could remove a space at the start of the typing. TINY-9310
 
 ### Added
+- New optional `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing tinymce table. #TINY-9808
```

Comment:
Dropping tinymce since all things are tinymce.

```suggestion
- New optional `table_merge_content_on_paste` option which disables the merging behaviour when pasting a table inside an existing table. #TINY-9808
```